target/classes/liquibase/scripts/notification_task.sql

```diff
@@ -0,0 +1,10 @@
+-- liquibase formatted sql
+
+-- changeset yarek2003:1
+create table notification_task(
+                                  notification_task_id serial primary key ,
```

Comment:
```suggestion
                                  notification_task_id bigserial primary key ,
```
тип должен быть bigserial, так как в сущности используешь Long